Add description about PessimisticLockNotFound relating to lazy uniqueness check  area/transaction Indicates that the Issue or PR belongs to the area of transaction (#10982)
pingcap/docs: constraints.md

    ERROR 9007 (HY000): Write conflict, txnStartTS=435688780611190794, conflictStartTS=435688783311536129, conflictCommitTS=435688783311536130, key={tableID=74, indexID=1, indexValues={bill, }} primary={tableID=74, indexID=1, indexValues={bill, }}, reason=LazyUniquenessCheck [try again later]
    ```

> - When this variable is disabled, if there is a write conflict among multiple pessimistic transactions, the pessimistic lock might be forced to roll back when other pessimistic transactions are committed, thus resulting in a `Pessimistic lock not found` error. When this error occurs, it means that deferring the unique constraint check of the pessimistic transaction is not suitable for your application scenario. In this case, consider adjusting the application logic to avoid the conflict or retrying the transaction.

Reviewer: ```suggestion
- When this variable is disabled, if there is a write conflict among multiple pessimistic transactions, the pessimistic lock might be forced to roll back when other pessimistic transactions are committed, thus resulting in a `Pessimistic lock not found` error. When this error occurs, it means that deferring the unique constraint check of the pessimistic transaction is not suitable for your application scenario. In this case, consider adjusting the application logic to avoid the conflict or retrying the transaction after an error occurs.
```

> - When this variable is disabled, if there is a write conflict among multiple pessimistic transactions, the pessimistic lock may be forced to roll back when the other pessimistic transactions commit, thus generating a `Pessimistic lock not found` error. When this error occurs, it means that the application scenario is not suitable for deferring the unique constraint check of the pessimistic transaction, and you should consider adjusting the application logic to avoid the conflict or retrying the transaction after the error occurs.

Reviewer 2: ```suggestion
- When this variable is disabled, if there is a write conflict among multiple pessimistic transactions, the pessimistic lock might be forced to roll back when other pessimistic transactions are committed, thus resulting in a `Pessimistic lock not found` error. When this error occurs, it means that deferring the unique constraint check of the pessimistic transaction is not suitable for your application scenario. In this case, consider adjusting the application logic to avoid the conflict or retrying the transaction after getting the error.
```
